### ISSUE TITLE
LayerTypeEAPOLKey Has No CanDecode()

### ISSUE THREAD

**Issue by Galadros** [google/gopacket#621]
A layer type for EAPOL keys- `LayerTypeEAPOLKey`- is defined in `eapol.go`, with such methods as `decodeEAPOLKey`, `DecodeFromBytes`, and `NextLayerType` all defined.  However, it doesn't have any `CanDecode()` method.  This means it can't be used as a `DecodingLayer`, essentially rendering the previous functions useless if you're parsing as shown in `parsers.go`.  Is there any reason not to simply add the following?

```
func (ek *EAPOLKey) CanDecode() gopacket.LayerType {
	return LayerTypeEAPOLKey
}
```
which is in line with all the other `CanDecode()` functions such as this for `LayerTypeEAPOL`:

```
func (e *EAPOL) CanDecode() gopacket.LayerClass {
	return LayerTypeEAPOL
}
```

It's quite possible that I'm not seeing some reason for things to be this way, but as-is a parser that proceeds through all the layers in a packet will break when it hits an EAPOL key layer.

EDIT: edited to improve text formatting.

**Comment by notti** [comment]
Looks to me like the author of the eapol key layer just forgot to add the method in 940451764f073c1bca137b5252f6a3f65fb07d4f

**Comment by notti** [comment]
Fixed by #635
